### ISSUE TITLE
do not re-raise every exception from main()

### ISSUE THREAD

**Issue by mattoufoutu** [PokemonGoF/PokemonGo-Bot#3125]
### Expected Behavior

When an error occurs, the stacktrace should show the right code section where it happened.

### Actual Behavior

Whatever unexpected error occurs, the stacktrace shows the `main()` function as the erroneous code location.

```
Traceback (most recent call last):
  File "pokecli.py", line 502, in <module>
    main()
  File "pokecli.py", line 133, in main
    raise e
WhateverError: any unexpected error occured, I don't know where it happened
```

### Steps to Reproduce

Run the bot until an error occurs.

### Other Information
OS: Linux 4.6.4 (ArchLinux)
Git Commit: ae0ae815089bf1e3b17497b476d524a0468949e2
Python Version: 2.7.12

**Comment by pmquan** [comment]
I believe it's already fixed in PR #2944

**Comment by mattoufoutu** [comment]
you're right, I didn't test the dev branch before posting, my bad.
